code/modules/modular_computers/file_system/programs/generic/ntnrc_client.dm

```diff
@@ -29,6 +29,11 @@
 	if(!comp)
 		return
 
+/datum/computer_file/program/chatclient/Destroy()
+	if(src in ntnet_global.chat_clients)
```

Comment:
Don't need this check. If it isn't in `ntnet_global.chat_clients` then `-=` won't remove it.

Reply:
monke moment